lib/sie/document.rb

```diff
@@ -89,17 +104,18 @@ def add_voucher(opts)
           account_number = line.fetch(:account_number)
           amount         = line.fetch(:amount)
           booked_on      = line.fetch(:booked_on)
+          dimensions     = line.fetch(:dimensions, {}).to_a.flatten
```

Comment:
No big deal – perhaps it's clearer the way it is – but you can skip `to_a` if `dimensions` is always a `Hash` or `Array`. Hashes can be flattened.